docs/how-to-debug-a-hyperlight-guest.md

```diff
@@ -201,3 +201,222 @@ involved in the gdb debugging of a Hyperlight guest running inside a **KVM** or
      └─┘                       │          |                                           |                                 |      │
       |                        └───────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+
+## Dumping the guest state to an ELF core dump when an unhandled crash occurs
+
+When a guest crashes because of an unknown VmExit or unhandled exception, the vCPU state is dumped to an `ELF` core dump file.
+This can be used to inspect the state of the guest at the time of the crash.
+
+To make Hyperlight dump the state of the vCPU (general purpose registers, registers) to an `ELF` core dump file, set the feature `crashdump` and run a debug build.
```

Comment:
Am I correct that is requires both the cargo feature, and also the runtime configuration to be set to true? If yes, then maybe update this